packages/flutter_fuchsia/lib/src/flutter_views.dart

```diff
@@ -0,0 +1,187 @@
+// Copyright 2018 The Fuchsia Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:collection';
+import 'dart:io';
+
+import 'package:process/process.dart';
+import 'package:logging/logging.dart';
+
+import 'dart/fuchsia_dart_vm.dart';
+import 'fuchsia_device_command_runner.dart';
+
+final String ipv4Loopback = InternetAddress.LOOPBACK_IP_V4.address;
+
+final Logger _log = new Logger('flutter_fuchsia::flutter_views');
+
+final ProcessManager _processManager = new LocalProcessManager();
+
+/// Persistent VM service cache to avoid repeating handshakes across function
+/// calls.
+final HashMap<int, FuchsiaDartVm> _fuchsiaDartVmCache =
+    new HashMap<int, FuchsiaDartVm>();
+
+/// Returns a list of flutter views for the given [ipv4Address].
```

Comment:
nit: Flutter style guide (deliberately in contradiction to the upstream dart style) uses backticks to refer to parameters rather than square brackets.

Reply:
Done.